package.json

```diff
@@ -20,7 +20,13 @@
     "dependencies": {
         "cookieconsent": "^3.1.0",
         "dotenv": "^7.0.0",
-        "node-sass-chokidar": "^1.3.3",
-        "react-scripts": "^1.1.5"
-    }
-}
+        "node-sass-chokidar": "^1.3.4",
+        "react-scripts": "^2.1.8"
+    },
+    "browserslist": [
+        ">0.2%",
+        "not dead",
+        "not ie <= 11",
```

Comment:
do we want to support older browsers?

Reply:
new react scripts blocks them by default

Reply:
@melikesofta I struggle to imagine a developer on something older than IE 11. That is indeed a default `browserslist` as provided by new react-scripts, so I consider it good enough until proven otherwise.

Reply:
@kojoru ack/go :D